### PR TITLE
Fixing Message Corrupt error in Playwright tests

### PR DESCRIPTION
**Pull request by ytimocin**
Adding a function that checks if the web application is ready to serve before running the Playwright tests.

### CODE REVIEW

File: playwright/playwright.config.ts [hunk 1]
```diff
@@ -20,31 +20,25 @@ export default defineConfig({
...
+      use: { ...devices["Desktop Chrome"] },
     },
-
-    {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
-    },
-
     {
-      name: 'webkit',
```
Comment (ytimocin) [inline playwright/playwright.config.ts]: https://github.com/microsoft/playwright/blob/main/tests/page/page-event-network.spec.ts#L62

File: playwright/tests/util/helper.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+export async function waitForWebApp(url: string | undefined, timeout = 30000) {
+  if (!url) {
+    throw new Error("URL is not defined");
+  }
+
+  const startTime = Date.now();
+  while (true) {
```
Comment (AaronCrawfis) [inline playwright/tests/util/helper.ts:9]: What will happen if the app never becomes ready? Will this fall back to the timeout of the GitHub Actions step?

Comment (ytimocin) [reply playwright/tests/util/helper.ts:9]: It is going to throw an error based on the timeout defined in the signature of the function: https://github.com/radius-project/samples/pull/978/files#diff-01141eafefa05f5a0809324840c2ed0088261d39f5fddd078b9be73baaac81a6R3.

Comment (AaronCrawfis) [reply playwright/tests/util/helper.ts:9]: Perfect, thanks!!

**Pull request closed without merging**